packages/vscode-graphql/package.json

```diff
@@ -160,7 +160,7 @@
   "devDependencies": {
     "@types/capitalize": "2.0.0",
     "@types/vscode": "1.62.0",
-    "esbuild": "0.15.10",
+    "esbuild": "0.18.10",
```

Comment:
@acao I updated esbuild everywhere because something was changed with the latest merged PRs and it brokes vite example to build 
![image](https://github.com/graphql/graphiql/assets/7361780/4aa7d218-0c04-4414-ab9d-01e9859c81b3)

Reply:
@acao eroor disappears when upgrading esbuild

> **Note**: vite under the hood uses esbuild, so I am pretty sure there are conflicts due to yarn hoisting

Reply:
I don't imagine there is an issue if both vite and esbuild are upgraded. vite never seems to lag far behind with esbuild versions it uses. we have no reason to use old versions of esbuild. i upgrade esbuild in the vscode e2e PR as well. we just need to manually test the vscode-graphql and vscode-graphql-exec bundles when changing these versions

Reply:
no problem now, seems to work fine now, and this won't be a problem after #3234